code-samples/messaging/send-mms.js

```diff
@@ -0,0 +1,70 @@
+/*You get the environment parameters from your 
+application dashbord in your developer account 
+https://developers.ringcentral.com/ */
+
+const RC = require('@ringcentral/sdk').SDK
+require('dotenv').config();
+

```

Comment:
Minor nit pick. But Let's eliminate unnecessary whitespace. There are numerous places where you have three newlines. Let's keep it to just two (a single blank line).